File: angulacommande/src/app/Model/commande-entete.ts
import { Client } from "./client";
import { CommandeLigne } from "./commande-ligne";

export class CommandeEntete {
    
    number=0;
    dateCmd: string="";
    numeroCmd: string="";
    montantTot: number=0;
    client: Client=new Client;
    idClt: number=0;
    commandeLignes: CommandeLigne[]=[];

}

File: Domain/Models/CommandeEntete.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Threading.Tasks;

namespace Domain.Models
{
    public class CommandeEntete
    {
        [Key]
        public int IdCmdEnt { get; set; }
        public DateTime DateCmd { get; set; }
        public string NumeroCmd { get; set; }
        public decimal MontantTot { get; set; }
        public Client Client { get; set; }
        public int IdClt { get; set; }
        public ICollection<CommandeLigne> CommandeLignes { get; set; }
    }
}
